app/code/Magento/SalesRule/Model/ResourceModel/Rule/DeleteCustomerUsage.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\SalesRule\Model\ResourceModel\Rule;
+
+
+class DeleteCustomerUsage
+{
+    /** @var \Magento\SalesRule\Model\ResourceModel\Rule\Customer */
+    protected $_customerRuleDetails;
+
+    /**
+     * DeleteCustomerUsage constructor.
+     * @param Customer $customerRuleDetails
+     */
+    public function __construct(
+        Customer $customerRuleDetails
+    )
+    {
+        $this->_customerRuleDetails = $customerRuleDetails;
+    }
+
+    /**
+     * Delete the time Usage from salesrule_customer table when times_used is 0
+     * @param int $ruleId
+     * @param int $customerId
+     * @param int $updatedTimeUsed
+     */
+    public function deleteCustomerTimeUsage($ruleId, $customerId, $updatedTimeUsed)
+    {
+        $connection = $this->_customerRuleDetails->getConnection();
+        if ($updatedTimeUsed == 0) {
```

Comment:
I suggest changing this statement according to the general rule of preferring the identical comparison operator `===` over the equal one `==`.

Reply:
@aleron75 ,
I have changed the operator from "==" to "===". kindly check and let me know.